handbook/company/pricing-features-table.yml

```
@@ -149,12 +149,12 @@
   moreInfoUrl: https://github.com/fleetdm/fleet/issues/5478
   productCategories: [Endpoint operations,Device management,Vulnerability management]
   pricingTableCategories: [Configuration]
-  usualDepartment: Security
-  tier: Premium
+  usualDepartment: IT
+  tier: Free
   jamfProHasFeature: yes
   jamfProtectHasFeature: yes
   waysToUse:
-    - description: Enforce email two-factor authentication (2FA) when logging in to Fleet for added security.
+    - description: Enforce two-factor authentication (2FA) when logging in to Fleet for added security.
```

Also revert this change to reduce diff in the PR to `main`:
```suggestion
    - description: Enforce two-factor authentication when logging in to Fleet for added security.
```